server/routes/overview/overviewView.ts

```diff
@@ -2,69 +2,23 @@ import type {
   ActionPlanReviews,
   Assessment,
   FunctionalSkills,
-  InPrisonCourse,
+  InductionSchedule,
   InPrisonCourseRecords,
   PrisonerGoals,
   PrisonerSummary,
 } from 'viewModels'
-import { isAfter, isWithinInterval, startOfToday, sub } from 'date-fns'
+import { startOfToday, sub } from 'date-fns'
 import type { InductionDto } from 'inductionDto'
+import type { OverviewViewRenderArgs } from './overviewViewTypes'
 import dateComparator from '../dateComparator'
-import ActionPlanReviewStatusValue from '../../enums/actionPlanReviewStatusValue'
-
-type RenderArgs = {
```

Comment:
This type, and the new types added for this PR, have been moved to `overviewViewTypes.ts`